server/routes/contact.py

```diff
@@ -0,0 +1,118 @@
+import logging
+import os
+import re
+import threading
+
+from dotenv import load_dotenv
+from email_validator import validate_email, EmailNotValidError
+from flask import Blueprint, request, jsonify
+from flask_restful import Api, Resource
+
+from server.utils.mail_templates import send_contact_email
+
+logging.basicConfig(
+    level=logging.DEBUG,
+    format="%(asctime)s [%(levelname)s] %(message)s"
+)
+logger = logging.getLogger(__name__)
+
+# Create the blueprint
+contact_bp = Blueprint('contact', __name__, url_prefix="/api")
```

Comment:
No need to prefix `/api` here, it is already registered in the `routes/__init__.py`